<launch>
    <node pkg="wilson_ros" type="zone_generator" name="ZoneGenerator" output="screen"/>
</launch>
